### ISSUE TITLE
Set env var per host in yml file

### ISSUE THREAD

**Issue by dpfahlbusch** [deployphp/deployer#1696]
| Q                 | A
| ----------------- | ---
| Issue Type        | Question
| Deployer Version  | 6.3
| Local Machine OS  | Win10
| Remote Machine OS | N/A

### Description 
Is there a possibility to set env vars in the hosts yml for each host?

for example:
dev host:
has a variable with develop context

prod host:
has a variable with production context

**Comment by notflip** [comment]
@antonmedv Use env: where?

**Comment by antonmedv** [comment]
In yaml

**Comment by agmckee** [comment]
An example of this usage would be helpful, I've tried it but it doesn't seem to work.

I've tried YAML lists:

`env:
    - foo: bar`

... and I've tried a standard dictionary under the env key.

I've searched the issues here and the documentation and can't find an example of the YAML syntax for env, other than your brief example [above](https://github.com/deployphp/deployer/issues/1696#issuecomment-419723339).
